.azure/pipelines/publish-infracost-setup-task.yml

```diff
@@ -16,16 +16,29 @@ stages:
             inputs:
               version: 'v0.7.x'
           - task: Npm@1
+            displayName: Install InfracostSetup dependencies
             inputs:
               command: 'install'
-              workingDir: 'tasks/setup/InfracostSetup'
+              workingDir: 'tasks/InfracostSetup'
           - task: Bash@3
-            displayName: Compile Javascript
+            displayName: Compile InfracostSetup Javascript
             inputs:
               targetType: 'inline'
               script: |
-                cd tasks/setup/InfracostSetup
-                tsc
+                cd tasks/InfracostSetup
+                npm run build
```

Comment:
@hugorut FYI, changed it to `npm run build`.

Reply:
Sounds good